lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/usecase/SaveExpenseTest.kt

```diff
@@ -22,4 +25,13 @@ public class SaveExpenseTest {
         val savedExpense = expenseContract.getExpense(expense.id)
         assertThat(savedExpense).isEqualTo(expense)
     }
+
+    @Test
+    public fun `verify that saveExpense usecase calls expenseContract`(): Unit = runBlockingTest {
+        val expense = DummyData.expense
+        val mockExpenseContract = mock(ExpenseContract::class.java)
```

Comment:
You don't need to add mock to the variable name.
Give it the name of the actual object

Reply:
I will change the name. I prefixed the mock to the variable name because it is a mocked object, and to also distinguish it from the expenseContract member variable